### ISSUE TITLE
[FR]: Options to disable article enclosures

### ISSUE THREAD

**Issue by RetroAbstract** [martinrotter/rssguard#1025]
### Brief description of the feature request

This feature request comes following [this explanation of article enclosures](https://github.com/martinrotter/rssguard/issues/1024#issuecomment-1653482232) **and has a different desired result than #1024**, that one being to modify the quality of image enclosures by changing elements of the image enclosure URLs, and this one being to disable them.

This feature would allow the user to disable article enclosures through:

- Article filters, for easy multi-feed enclosure disabling.
- Option in: "Settings > Feeds & Articles > Articles > **Disable article enclosures**", for easy enclosure disabling globally.
- Option when right clicking a feed: "Edit selected item > Articles > **Disable article enclosures**", for easy enclosure disabling for one feed.

Thanks

**Comment by martinrotter** [comment]
There already is possibility of disabling display of inline article enclosures: Articles -> "Display attachments directly...:" in settings.

**Comment by RetroAbstract** [comment]
> There already is possibility of disabling display of inline article enclosures: Articles -> "Display attachments directly...:" in settings.

True, forgot that. However this doesn't allow you to filter which articles you'd like to have enclosures displayed in and articles you wouldn't.

**Comment by martinrotter** [comment]
yes, that is true

if you wish to actully remove enclosures from incoming article, that would have to be added

**Comment by martinrotter** [comment]
Is now possible via article filtering function `msg.removeAllEnclosures();`.
